libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_receive.c

```diff
@@ -1217,15 +1281,15 @@ TEST( MQTT_Unit_Receive, PubackInvalid )
         TEST_ASSERT_EQUAL_INT( true, _processBuffer( &publish,
                                                      pPuback,
                                                      pubackSize,
-                                                     IOT_MQTT_BAD_RESPONSE ) );
+                                                     IOT_MQTT_STATUS_PENDING ) );
 
         /* Network close should have been called for invalid packet. */
         TEST_ASSERT_EQUAL_INT( true, _networkCloseCalled );
         TEST_ASSERT_EQUAL_INT( true, _disconnectCallbackCalled );
         _networkCloseCalled = false;
         _disconnectCallbackCalled = false;
     }
-
+    IotListDouble_RemoveHead( &( _pMqttConnection->pendingResponse ) );
```

Comment:
What's the reason for this new line?

Reply:
Provided the comment for this addition in the code.